src/components/airtable/Airtable.tsx

```diff
@@ -0,0 +1,14 @@
+import Airtable from 'airtable';
+
+const readOnlyApiKey = 'key2a9aPpR6jDvtau';
+const appId = 'appKoO9jQuqptnJNk';
+const base = new Airtable({ apiKey: readOnlyApiKey }).base(appId);
+
+export const FetchRestaurants = (query) => {
```

Comment:
This is awesome! Let's move this file to `src/services` or something since it's not really a React component.